src/Board_test.py

```diff
@@ -80,9 +81,77 @@ def test_adj(self):
             start=(2, 3), terrain_filter=Terrain.MOUNTAIN, shipping_limit=1
         ) == [(4, 2)]
 
-        assert aMap.check_adjacency((7, 7), (9, 6), 0) == "Not"
-        assert aMap.check_adjacency((7, 7), (9, 6), 1) == "Indirect"
-        assert aMap.check_adjacency((7, 7), (8, 3), 2) == "Not"
-        assert aMap.check_adjacency((7, 7), (8, 3), 3) == "Indirect"
-        assert aMap.check_adjacency((7, 7), (8, 1), 4) == "Not"
-        assert aMap.check_adjacency((7, 7), (8, 1), 5) == "Indirect"
+        assert aMap.check_adjacency((7, 7), (9, 6), 0) == None
+        assert aMap.check_adjacency((7, 7), (9, 6), 1) == AdjacencyType.Indirect
+        assert aMap.check_adjacency((7, 7), (8, 3), 2) == None
+        assert aMap.check_adjacency((7, 7), (8, 3), 3) == AdjacencyType.Indirect
+        assert aMap.check_adjacency((7, 7), (8, 1), 4) == None
+        assert aMap.check_adjacency((7, 7), (8, 1), 5) == AdjacencyType.Indirect
+
+    def test_start(self):
+        aMap = Board()
+        assert aMap.get((2, 3))._terrain == Terrain.DESERT
+        assert aMap.start(Terrain.LAKE) == [
```

Comment:
If we change the function to return the `Tile` objects, this could become something like:
```suggestion
        assert sum(int(tile.terrain != Terrain.LAKE) for tile in aMap.start(Terrain.LAKE)) == 0
```